### PR TITLE
add EMBOSS Sixpack tutorial

### PR DESCRIPTION
**Pull request by alstonlo**
This PR adds a tutorial for the EMBOSS Sixpack tools.

### CODE REVIEW

File: tools/img/sixpack/contig.fasta [hunk 1]
```diff
@@ -0,0 +1,184 @@
+>NODE_133
```
Comment (ababaian) [inline tools/img/sixpack/contig.fasta]: Could you add a more descriptive name please?

File: tools/sixpack.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+right to left for all intents and purposes. For example, the `S` in the following is produced by the 
+codon `TCC`, not `CCT` or `TGT`:
+
+```
+1 GTGTCCTGTCTTGTTGTGTTTAAACCGTCCTTAGTGTACTATTGCCAAGTGCGAGAAATC 60     (L6)
+        |                                                             (L7 omitted)
+  X  C  S  L  V  V  C  I  Q  C  S  D  C  S  L  P  E  R  E  K     F5   (L8)
+```
+
+Finally, at the very end of the page, there will also be some summary statistics that count
```
Comment (ababaian) [inline tools/sixpack.md:122]: So is more ORFs better or less ORFs better? How are you interpreting these values?

**Review by ababaian — COMMENTED** [review]
Great tutorial, I like the use of inline code-blocks here.

File: tools/img/sixpack/contig.fasta (hunk 1 above)
Comment (alstonlo) [reply tools/img/sixpack/contig.fasta]: I've added the SRA run name as a prefix!

File: tools/sixpack.md (hunk 2 above)
Comment (alstonlo) [reply tools/sixpack.md:122]: Honestly, I am not interpreting these values as I am not sure how to. I was unable to find anything in my literature search either.

Comment (ababaian) [reply tools/sixpack.md:122]: OK consider the example with SARS-CoV-2 in class:

SC2 ORF Calling

This won't be in the literature / textbook, you have to assess it yourself. What does it mean if a particular frame has very few ORFs, and what does it mean if a frame has a very high number of ORFs. How does ORF length vary with each of these?

Comment (alstonlo) [reply tools/sixpack.md:122]: If a frame has few (resp., many) ORFs, then the ORF lengths in that frame will be larger (resp., smaller). From my understanding, longer ORFs are favoured in analyses. However, I feel hesitant to use this analysis as it fails when multiple long ORFs exist in different frames. Instead, I would prefer to sort the entries in `Tool Output` by length.

**Pull request merged**